src/main/scala/com/gd/twitterstreamingtToDatalake/TransformTweets.scala

```diff
@@ -6,19 +6,18 @@ import twitter4j.Status
 object TransformTweets {
 
   def getText(englishTweets: DStream[Status]): DStream[String] = {
-    val text = englishTweets.flatMap(x => x.getText.split(" "))
+    val text = englishTweets.map(x => x.getText)
```

Comment:
JFYI: 
a more consise yet reasonable version 
```scala

englishTweets.map(_.getText)

```

Reply:
Got it !